src/main/java/org/snomed/snowstorm/rest/BranchController.java

```diff
@@ -131,6 +132,26 @@ public void unlockBranch(@PathVariable String branch) {
 		branchService.unlock(branch);
 	}
 
+	@RequestMapping(value = "/branches/{branchPath}/actions/set-author-flag", method = RequestMethod.POST)
+	@PreAuthorize("hasPermission('AUTHOR', #branchPath)")
+	public BranchPojo setAuthorFlag(@PathVariable String branchPath, @RequestBody SetAuthorFlag setAuthorFlag) {
```

Comment:
This method will receive a key/value and update the Branch's metadata accordingly.